### DISCUSSION TITLE
load: PageServerLoad (event and params)

### DISCUSSION THREAD

**Discussion by Tihomir971** [sveltejs/kit#7538] [Q&A] [answered]
I have load functions with event and it's workin and with params and it's working.
How can I call load function with event and param at same time? I get error when try to pass them both.

**Comment by david-plugge** [comment]
Can you give an example on what you are trying to do?

**Comment by teemingc** [comment, marked as answer]
`params` is a part of the `event` object, e.g., `event.params`
You could also extract params from event.
```ts
export const load = (event) => {
  // use can use event
  const { params } = event;

  // you can use params or event.params
}
```
More info on what's in `event` in the type docs https://kit.svelte.dev/docs/types#sveltejs-kit-requestevent
